src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -251,10 +249,7 @@ private void setLookAndFeel() {
             if (UIManager.getCrossPlatformLookAndFeelClassName().equals(lookFeel)
                     && !System.getProperty("java.runtime.name").contains("OpenJDK")) {
                 // try to avoid ending up with the ugly Metal L&F
-                Plastic3DLookAndFeel lnf = new Plastic3DLookAndFeel();
-                MetalLookAndFeel.setCurrentTheme(new SkyBluer());
-                com.jgoodies.looks.Options.setPopupDropShadowEnabled(true);
-                UIManager.setLookAndFeel(lnf);
+                UIManager.setLookAndFeel(new NimbusLookAndFeel());
```

Comment:
you should set the LookAndFeel based on className, as recommend by the jdk api:
   UIManager.setLookAndFeel("javax.swing.plaf.metal.MetalLookAndFeel");

Reply:
Thanks for pointing out! Changed it.